README.md

```diff
@@ -38,70 +38,95 @@ The second: you need not only a secure context for the browser, but actual safet
 
 ## Overview
 
-1. Get a server. It doesn't need to be big. Ideally you should have at least one slave, too, because NS entries require at least two servers.
-1. Point the NS entry of your domain to this server.
-1. [Install deps](#base-installation-and-deps).
-1. [Run dnsserver.py](#running-the-dns-server).
-1. [Create the certificates running certbotdns.py](#renewing-keys).
+1. Get an Ubuntu server. It doesn't need to be big. Ideally you should have at least one slave, too, because NS entries require at least two servers.
```

Comment:
There's no need to restrict this to Ubuntu. I ran this in other flavors of Linux.

Reply:
Fair!  With the install script being `apt` based, we'll at least need Debian, but I agree we can open it up and leave that script for Ubuntu/Debian and encourage all other flavors. 

Fixed in 0870541